README.md

```diff
@@ -94,12 +94,12 @@ export GITACCESS=<YOUR_TOKEN>
 
 Run the script by passing in `organization` and `repo`  
```

Comment:
Since `repo` is optional now, I would suggest removing `repo` from here or show both options. Something like this:

Run the script to pull data from all repositories in an organization 
```python
python ./ml-conversational-analytic-tool/githubDataExtraction.py <organization>
```

Run the script to pull data from a repository in an organization 
```python
python ./ml-conversational-analytic-tool/githubDataExtraction.py <organization> --repo <repo>
```

Reply:
Done